example_external/README.md

```diff
@@ -0,0 +1,155 @@
+# Example:
```

Comment:
This probably could use a more descriptive title showing this is for how to include parameters from another library, as well as a sentence or two describing things before jumping right into instructions.